pkg/prow/job.go

```diff
@@ -231,3 +209,31 @@ func (j *Job) CreateChangedCovHtml() error {
 	}
 	return err
 }
+
+func getFilesAndCovList(fullDiff bool, prComment github.PrComment, localP, baseP cover.CoverageList) (changedFiles []string, deltaCovList cover.DeltaCovList, err error) {
+	if !fullDiff {
+		// get github pull request changed files' name
+		var ghChangedFiles, err = prComment.GetPrChangedFiles()
+		if err != nil {
+			logrus.Errorf("Get pull request changed file failed.")
+			return nil, nil, err
+		}
+		if len(ghChangedFiles) == 0 {
+			logrus.Printf("0 files changed in github pull request, don't need to run coverage profile in presubmit.\n")
+			return nil, nil, nil
+		}
+		changedFiles = trimGhFileToProfile(ghChangedFiles)
+
+		// calculate diff cov between local and remote profile
+		deltaCovList = cover.GetChFileDeltaCov(localP, baseP, changedFiles)
+		return changedFiles, deltaCovList, nil
+	}
+	deltaCovList = cover.GetDeltaCov(localP, baseP)
+	logrus.Infof("get delta file name is:")
+	for _, d := range deltaCovList {
+		logrus.Infof("%s", d.FileName)
+		changedFiles = append(changedFiles, d.FileName)
+	}
+
```

Comment:
if necessary, it can print a information here

Reply:
add info before return